heron/tracker/src/python/handlers/containerfilehandler.py

```diff
@@ -60,6 +62,7 @@ class ContainerFileStatsHandler(BaseHandler):
   Parameters:
    - cluster - Name of cluster.
    - environ - Running environment.
+   - role - Name of person who submits the topology (optional)
```

Comment:
not limited to person, sometimes people may use their service account to submit a topology.

`role used to submit the topology (optional)` is good enough